explain-walkthrough.md

```diff
@@ -0,0 +1,206 @@
+4 rows in set (0.00 sec)
+```
+
+To compare the actual execution time, we can again use [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md):
+
+{{< copyable "sql" >}}
+
+```sql
+EXPLAIN ANALYZE SELECT count(*) FROM trips WHERE start_date BETWEEN '2017-07-01 00:00:00' AND '2017-07-01 23:59:59';
+```
+
+```sql
++-----------------------------+---------+---------+-----------+-------------------------------------------+------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------+-----------+------+
+| id                          | estRows | actRows | task      | access object                             | execution info                                                                                                   | operator info                                                     | memory    | disk |
++-----------------------------+---------+---------+-----------+-------------------------------------------+------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------+-----------+------+
+| StreamAgg_17                | 1.00    | 1       | root      |                                           | time:4.516728ms, loops:2                                                                                         | funcs:count(Column#13)->Column#11                                 | 372 Bytes | N/A  |
+| └─IndexReader_18            | 1.00    | 1       | root      |                                           | time:4.514278ms, loops:2, cop_task: {num: 1, max:4.462288ms, proc_keys: 11409, rpc_num: 1, rpc_time: 4.457148ms} | index:StreamAgg_9                                                 | 238 Bytes | N/A  |
+|   └─StreamAgg_9             | 1.00    | 1       | cop[tikv] |                                           | time:4ms, loops:12                                                                                               | funcs:count(1)->Column#13                                         | N/A       | N/A  |
+|     └─IndexRangeScan_16     | 8471.88 | 11409   | cop[tikv] | table:trips, index:start_date(start_date) | time:4ms, loops:12                                                                                               | range:[2017-07-01 00:00:00,2017-07-01 23:59:59], keep order:false | N/A       | N/A  |
++-----------------------------+---------+---------+-----------+-------------------------------------------+------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------+-----------+------+
+4 rows in set (0.00 sec)
+```
+
+The query time has reduced from 1.03 seconds to 0.0 seconds. Not bad!
+
+> **Tip:**
```

Comment:
```suggestion
> **Note:**
```

Reply:
I think this is a tip not a note, since it is tangential to the article.

Reply:
Only `Note` can be correctly parsed by the docs page builder as a note format. For example: 
![image](https://user-images.githubusercontent.com/41534398/94092956-993c6e00-fe4e-11ea-92a5-7f1f3085c12d.png)